src/app/utils/DiscordUtils.ts

```diff
@@ -154,13 +155,29 @@ const DiscordUtils = {
             await commandContext.send({ content: content, ephemeral: true });
             // await commandContext.delete();
         } else {  // This was a reaction or a DB event
-            await toUser.send(content);
+            try {
+                await toUser.send(content);
+            } catch (e) {
+                throw new AuthorizationError(
+                    `Thank you for giving bounty commands a try!\n` +
+                    `It looks like bot does not have permission to DM you.\n` +
+                    'Please give bot permission to DM you and try again.'
```

Comment:
The error message makes it sound like the bounty action aborted. It might or might not depending on the situation I think. So this probably needs to bubble up and the message adjusted at the caller.